packages/generator-react-server/test/app.js

```diff
@@ -24,6 +24,46 @@ test('generator-react-server:app creates default files', async t => {
 	t.false(await exists('docker-compose.yml', testDir));
 });
 
+test('generator-react-server:app .reactserverrc matches react-server-cli default option keys', async t => {
+	let testDir;
+	await helpers.run(path.join(__dirname, '../generators/app'))
+		.inTmpDir(dir => {
+			testDir = dir;
+		})
+		.withPrompts({name: 'foo', dockerCfg: false})
+		.toPromise();
+
+	// Reading files here instead of requiring because we don't understand export default here.
+	const defaultOptions = await readFile('defaultOptions.js', path.join(__dirname, '../../react-server-cli/src'));
```

Comment:
I get this error when I try and import the file at the top:

`import defaultOptions from '../../react-server-cli/src/defaultOptions.js';`

```
TAP version 13
/Users/jason/Projects/react-server/packages/react-server-cli/src/defaultOptions.js:14
export default {
^^^^^^
SyntaxError: Unexpected token export
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:513:28)
```